### ISSUE TITLE
[Feature Request ] Making keyboard more friendly 

### ISSUE THREAD

**Issue by teynav** [AnySoftKeyboard/AnySoftKeyboard#2364]
### Toggle or changes to make 
1.Why does h key has underscore that far away from hand and tilde those are more used :/
![20200706_110516](https://user-images.githubusercontent.com/40721108/86559032-8189d380-bf4a-11ea-8f32-bad51c56a696.jpg)

2.that key beside l key which is quotes key,  I hate it,  can we have toggle for that ? Its  not present in any keyboard i used before.
3. Ease the clutter on initial  page and move some symbols  to the back keep most used on front .
4. A toggle  to add or remove accented characters. 

**List of installed add-ons (like languages, or themes):**

**Comment by Tomatoide** [comment]
I agree on the underscore issue it's the most annoying thing about this keyboard imo
